web-client/src/ustc-ui/Icon/CaseIcons.tsx

```diff
@@ -5,8 +5,11 @@ import classNames from 'classnames';
 
 export function CaseIcons({ formattedCase }: { formattedCase: any }) {
   return (
-    <div className="multi-filing-type-icon">
-      <div
+    <div
+      className="multi-filing-type-icon"
+      style={{ display: 'flex', flexWrap: 'nowrap', gap: '1rem' }} // USWDS overwrites these styles at certain breakpoints even with !important
```

Comment:
I cannot find a better way to do this. At a certain breakpoint, USWDS's styles overwrite the styles in `multi-filing-type-icon`. This seemed liked the most practical workaround.